src/components/AnonymousChat/child/ChannelImage.tsx

```diff
@@ -200,7 +240,10 @@ const ChannelImage = ({
     return (
       <View>
         {renderMainImage()}
-        <FastImage source={AnonymousProfile} style={styles.postNotificationImage} />
+        <FastImage
+          source={AnonymousProfile}
+          style={[styles.postNotificationImage, styles.anonPmNotificationImageContainer]}
+        />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The style of the `FastImage` component has been updated. Ensure that these changes are reflected across the application for consistency and do not affect readability or accessibility.
```diff
- <FastImage source={AnonymousProfile} style={styles.postNotificationImage} />
+ <FastImage
+   source={AnonymousProfile}
+   style={[styles.postNotificationImage, styles.anonPmNotificationImageContainer]}
+ />
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->